src/dash-table/dash/DataTable.js

```diff
@@ -19,1362 +12,22 @@ import LazyLoader from 'dash-table/LazyLoader';
-     * `derived_viewport_row_ids` contains IDs for the current page,
-     * while `derived_virtual_row_ids` contains IDs across all pages.
-     */
-    derived_virtual_row_ids: PropTypes.arrayOf(
-        PropTypes.oneOfType([PropTypes.string, PropTypes.number])
-    ),
-
-    /**
-     * `derived_virtual_selected_rows` represents the indices of the
-     *  `selected_rows` from the perspective of the `derived_virtual_indices`.
-     */
-    derived_virtual_selected_rows: PropTypes.arrayOf(PropTypes.number),
-
-    /**
-     * `derived_virtual_selected_row_ids` represents the IDs of the
-     * `selected_rows` as they appear after filtering and sorting,
-     * across all pages.
-     */
-    derived_virtual_selected_row_ids: PropTypes.arrayOf(
-        PropTypes.oneOfType([PropTypes.string, PropTypes.number])
-    ),
-
-    /**
-     * Object that holds the loading state object coming from dash-renderer
-     */
-    loading_state: PropTypes.shape({
-        /**
-         * Determines if the component is loading or not
-         */
-        is_loading: PropTypes.bool,
-        /**
-         * Holds which property is loading
-         */
-        prop_name: PropTypes.string,
-        /**
-         * Holds the name of the component that is loading
-         */
-        component_name: PropTypes.string
-    }),
-
-    /**
-     * Used to allow user interactions in this component to be persisted when
-     * the component - or the page - is refreshed. If `persisted` is truthy and
-     * hasn't changed from its previous value, any `persisted_props` that the
-     * user has changed while using the app will keep those changes, as long as
-     * the new prop value also matches what was given originally.
-     * Used in conjunction with `persistence_type` and `persisted_props`.
-     */
-    persistence: PropTypes.oneOfType([
-        PropTypes.bool,
-        PropTypes.string,
-        PropTypes.number
-    ]),
+const DataTable = props => (
+    <Suspense fallback={null}>
+        <RealDataTable {...props} />
+    </Suspense>
+);
 
-    /**
-     * Properties whose user interactions will persist after refreshing the
-     * component or the page.
-     */
-    persisted_props: PropTypes.arrayOf(
-        PropTypes.oneOf([
-            'columns.name',
-            'data',
-            'filter_query',
-            'hidden_columns',
-            'selected_columns',
-            'selected_rows',
-            'sort_by'
-        ])
-    ),
+export default DataTable;
 
-    /**
-     * Where persisted user changes will be stored:
-     * memory: only kept in memory, reset on page refresh.
-     * local: window.localStorage, data is kept after the browser quit.
-     * session: window.sessionStorage, data is cleared once the browser quit.
-     */
-    persistence_type: PropTypes.oneOf(['local', 'session', 'memory'])
-};
+const RealDataTable = asyncDecorator(DataTable, () =>
+    LazyLoader.table().then(res => {
+        const realDataTable = res.default;
```

Comment:
`import(...)` loads like an oldschool `require` -> need to get the default export on the `default` prop